src/module.rs

```diff
@@ -118,6 +125,43 @@ impl<'a> fmt::Display for Module<'a> {
     }
 }
 
+fn ansi_strings_modified(ansi_strings: Vec<ANSIString>, shell: String) -> Vec<ANSIString> {
+    ansi_strings
+        .iter()
+        .map(|ansi| {
+            let mut escaped = false;
+            let final_string: String = ansi
+                .to_string()
+                .chars()
+                .map(|x| match x {
+                    '\u{1b}' => {
```

Comment:
My preference would be to have compile-time consts in the start of the function to avoid having magic constants here, so:

```
const ESCAPE_BEGIN : char = '\u{1b}';
const MAYBE_ESCAPE_END : char = 'm';
```
at the top of the function and then matching on `ESCAPE_BEGIN` and `MAYBE_ESCAPE_END` in the code. (Again, pick names that make sense to you).